pkg/controller/informers/factory.go

```diff
@@ -38,6 +38,7 @@ type SharedInformerFactory interface {
 	PersistentVolumes() PVInformer
 
 	DaemonSets() DaemonSetInformer
+	ServiceAccounts() ServiceAccountInformer
```

Comment:
nit: please order this with the other core resources.  under `PersistentVolumes()`